src/core/placeTooltip.js

```diff
@@ -197,11 +195,6 @@ export default function placeTooltip(
 
   arrowLayer.style.display = "inherit";
 
-  if (typeof helperNumberLayer !== "undefined" && helperNumberLayer !== null) {
```

Comment:
Why is this no longer needed?

Reply:
@bozdoz I changed the position of that element and we don't need to adjust it after every refresh. It's in the tooltip container now.